src/api/v1/motd.go

```diff
@@ -0,0 +1,18 @@
+package v1
+
+import (
+	"github.com/ImpactDevelopment/ImpactServer/src/util"
+	"github.com/labstack/echo"
+	"net/url"
+)
+
+const motdUrl = "https://raw.githubusercontent.com/ImpactDevelopment/Resources/master/data/motd.txt"
+
+func getMotd(c echo.Context) error {
+	target, err := url.Parse(motdUrl)
+	if err != nil {
+		return err //wtf
+	}
+	util.Proxy(c, target)
```

Comment:
i love util.Proxy it makes everything so much easier

Reply:
makes testing a pain though lol

Reply:
haha imagine testing lol!!!!!